plugins/CorePluginsAdmin/Menu.php

```diff
@@ -20,9 +20,18 @@ class Menu extends \Piwik\Plugin\Menu
 {
     private $marketplacePlugins;
```

Comment:
@tsteur as there are now such nice and complete UI  tests & fixture for the Marketplace, would it maybe be do-able to add a single UI test for this " counter-plugins-to-update displayed-in-menu" feature?